### DISCUSSION TITLE
Stable Version 2

### DISCUSSION THREAD

**Discussion by kuti-JP** [patrykandpatrick/vico#978] [Questions]
### Question

Hello, thanks for the great work put into this. I have been using the v2 since alpha with the intention that whenever v2 is stable I won't have a lot to changes from version v1 and also I can only use the stable version for my project. So I was wondering if there is timeframe when v2 will move to stable. 

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @kuti-JP.
> I can only use the stable version for my project

Could you clarify? Why can't you use the beta?

> So I was wondering if there is timeframe when v2 will move to stable.

We plan on going stable soon. However, this is a FOSS project that we develop in our free time, so we can't provide specific ETAs.

**Reply by kuti-JP** [reply]
> Could you clarify? Why can't you use the beta?

Just a product requirement.

> We plan on going stable soon. However, this is a FOSS project that we develop in our free time, so we can't provide specific ETAs.

The efforts are duly appreciated. Thanks
